.github/workflows/executor-tests.yaml

```diff
@@ -0,0 +1,51 @@
+name: "sequential-execution-performance"
+on:
+  pull_request:
```

Comment:
Lets add workflow_dispatch here as well, that will allow you to run it from outside a PR as well

Reply:
I'd do the following here:
* Create a reusable workflow called `workflow-run-executor-benchmark.yaml`, and have it triggerable `workflow_call`
* Then, in a separate workflow (the entrypoint, e.g. `executor-tests.yaml`), trigger the reusable workflow above via workflow call. `executor-tests.yaml` can then be triggered by either `pull_request(target)` (on PR), or `workflow_dispatch` (manually via UI or API/CLI)

For an example, check out how it's implemented for replay-verify:
* Here's the reusable workflow, which contains the meat of the logic for running the job: https://github.com/aptos-labs/aptos-core/blob/main/.github/workflows/workflow-run-replay-verify.yaml
* Here's the workflow that calls it: https://github.com/aptos-labs/aptos-core/blob/main/.github/workflows/replay-verify.yaml